.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,32 @@
+name: Feature Request
+description: A feature request for WildFly
+title: "[RFE]: "
+labels: ["feature"]
+projects: ["wildfly/7"]
```

Comment:
We may want to consider using a workflow to add the project if not everyone create an issue will have write access. See https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms.